### PR TITLE
Add travis deploy encrypted key

### PR DESCRIPTION
**Pull request by alogan519**

### CODE REVIEW

**Comment by alogan519** [comment]
!set-reviewers @dbatiste @mpharoah-d2l @phil519

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,13 @@
 language: node_js
 node_js:
 - '4.1'
+
+deploy:
+  provider: npm
+  email: d2ltravisdeploy@d2l.com
+  api_key:
+    secure: jVXxWnz33GR6sAZmUNZTJwhI5xEsSHojT9BpN+FbtTlv7VCOdtmxaNPbOEvDW4dnVNRaxQB8CQQgGu/xn//UE63H7Zehq2l4vmcQZF4CvMcsZc7V5nm/66XJXLJQgVIuM3WGmOw8NtK1i0xBVtCGSN51kMqq7yLVCYcExt5Owh84k2/yTu2kOZ/pYq5cqEIJrNGAy8VpBeDTDwMrw9BbJnYEXSvf1GvSktFXeXPsLOwf4ihJf7hZt0i3SnU7+KXCzLlrjSouhJKY9Zh+/jME3cBT7ssm1AXzNmsTe6uUr7HxTO37ORvYT5qnIztJFriiFqsfeoNE+mgNEEC2J9Faoo8oSMVsbEyHfJbqKhpfFbdiXjfqM6tRoj7IBgrajIsSDVeA2zwDFzhDDWJmNfqzgkemjMuDTYx5rZTX6kBGhq3DygVqX7AqTxMp/xWxtNy1Hqx2yQOooQvUYi0G7GdEkwqlwpk5ZhyhTStEy9TlH1+d/E8dc7IeqigFZO59bCCFVQScI5iyOwub4C16x+2EbHxWTki+tZ3B/psL009nN+dI5IKouYaNuK5U7xG0h7viA1CaL9HGtdJi5wth2fRiTJXFejgd5q4SMJI4mCi+glfGvUPVqmBPQLAxdkYRDq9C3DZSI0u00Gel2Wrm/VELgtm0I7yUPRr1Y83oxKUMm44=
```
Comment (mpharoah-d2l) [inline .travis.yml]: Is this file ignored by bower? We don't want this file being published.

Comment (dbatiste) [inline .travis.yml]: It should be included in the ignore section of bower.json.  Not worried about the key though since it's encrypted.

Comment (alogan519) [inline .travis.yml]: I wonder if I should regenerate the key since the repo was renamed

Comment (dbatiste) [inline .travis.yml]: Not sure.  If it fails to publish, it's easy enough to delete the tag/release and try again after updating the key.

Comment (alogan519) [inline .travis.yml]: This file is ingored by power since it's a . file (hidden) and covered under **/.*

**Comment by mpharoah-d2l** [comment]
:+1:

**Pull request merged**